hw/top_earlgrey/cdc/chip_earlgrey_asic_cdc_cfg.hjson

```diff
@@ -24,17 +24,11 @@
   // Timing constraints for this module
   sdc_file: "{proj_root}/hw/top_earlgrey/syn/chip_earlgrey_asic.sdc"
 
-  // Verix environment file with additional definitions (may have to populate later)
-  env_file: ""
-
   // Main CDC waiver file
-  cdc_waiver_file: "{proj_root}/hw/top_earlgrey/cdc/cdc_waivers.tcl"
-
-  // CDC customized env file for ast integration
-  cdc_user_env: "{proj_root}/hw/top_earlgrey/cdc/top_user.env"
+  cdc_waiver_file: ""
 
-  // AST.lib
-  ast_lib: "{proj_root}/hw/top_earlgrey/ip/ast/lib/ast.lib"
```

Comment:
Do you know what the purpose of this lib file was?

Reply:
This was a liberty model of AST, which isn't available in the open-source domain. As the CDC flow no longer uses closed-source models/code, this can now be removed